include/envoy/network/connection_handler.h

```diff
@@ -147,8 +154,8 @@ class ActiveUdpListenerFactory {
    * @return the ActiveUdpListener created.
    */
   virtual ConnectionHandler::ActiveListenerPtr
-  createActiveUdpListener(ConnectionHandler& parent, Event::Dispatcher& disptacher,
-                          Network::ListenerConfig& config) PURE;
+  createActiveUdpListener(uint32_t worker_id, ConnectionHandler& parent,
```

Comment:
nit: worker_index, here and anywhere else we still have id. Also needs doc comment update.